src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2823,6 +2909,22 @@ boolean isVerticalPositionRelativeToFontSize() {
         float getVerticalPosition() {
             return verticalPosition;
         }
+
+        @Override
+        public int hashCode() {
+            int hashCode = Float.hashCode(horizontalPosition);
+            hashCode |= Float.hashCode(verticalPosition);
+            hashCode |= Short.hashCode(relative);
+            return hashCode;
```

Comment:
It's a nit yet you can simply return the computed value:
```suggestion
            return Float.hashCode(horizontalPosition)
                   | Float.hashCode(verticalPosition)
                   | Short.hashCode(relative);
```